app/mailers/employee_invitation_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class EmployeeInvitationMailer < ApplicationMailer
+  default from: 'test@test.com'
+
+  def invite_employee(email)
+    mail(to: email,
+      subject: 'Comarev - Invitation to join in',
+      template_path: 'employee_invitation',
+      template_name: 'invite_employee')
```

Comment:
it can be removed if you fix the convention of your views folder, I think it must be named `employee_invitation_mailer`